luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -24,12 +24,15 @@ local collisionVelocityThreshold = 99 / Game.gameSpeed
 local validCollisionAngleMultiplier = math.cos(math.rad(20)) --degrees
 
 -- Decrease this value to make units move less from impulse. This defines the maximum impulse allowed, which is (maxImpulseMultiplier * mass) of each unit.
-local maxImpulseMultiplier = 165 / Game.gameSpeed
+local maxImpulseMultiplier = 5.5
 
--- elmo/s, converted to elmo/frame. If a unit is launched via explosion faster than this, it is instantly slowed to the derivitive velLengthOffsetCap.
+--to save performance and reduce unit hesitation from nominal impulse, impulse values below (maxImpulseMultiplier * mass) returns 0 impulse.
+local minImpulseMultiplier = 1
```

Comment:
Can I ask what the purpose of defining "1" is here? Seems it has one use, never changes, and so you're always multiplying a number by "1"

Reply:
Because if I just have a number somewhere that doesn't have a variable name or a comment associated with it, a future maintainer won't know why I selected that number. Also this ends up being a configurable static variable somebody who doesn't understand my gadget very well can look at, understand and tweak